examples/sqlserverflex/sqlserverflex.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/stackitcloud/stackit-sdk-go/core/config"
+	"github.com/stackitcloud/stackit-sdk-go/core/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex/wait"
+)
+
+func main() {
+	// Specify the project ID
+	projectId := "16f49d71-37ad-4137-8b97-44d9c55c4094"
+
+	// Specify instance configuration options
+	// flavorId := "FLAVOR_ID"
+	// version := "VERSION"
```

Comment:
I think we can uncomment these and delete the actual flavor and version below. This way we dont induce errors if the flavors/versions become deprecated.

Reply:
Same as above